### ISSUE TITLE
Qdrant: Retrieve an existing collection

### ISSUE THREAD

**Issue by miary** [deepset-ai/haystack-core-integrations#729]
All examples showing integration of Qdrant with haystack show how to create a new Qdrant index/collection and use that as a DocumentStore. 

However, there does not seem to be a way to use an already-created Qdrant collection and use that within a Haystack project.

Please let me know if such a method exists.

**Comment by anakin87** [comment]
Sorry for the late response.

Since we want to provide a consistent user experience with different databases, in general our integrations are a bit opinionated.

To be used in Haystack, the points of your Qdrant collection must be convertible by [this method](https://github.com/deepset-ai/haystack-core-integrations/blob/1f71c721049f91634a96d61c5b505e66649655c5/integrations/qdrant/src/haystack_integrations/document_stores/qdrant/converters.py#L64)

In short, apart from the vector (which is mapped to `embedding`), the attributes should reflect those of the [`Document` dataclass](https://docs.haystack.deepset.ai/docs/data-classes#attributes-6).

If your collection is different, you might consider creating a migration script.

**Comment by anakin87** [comment]
An example of a migration script (in this case to add a sparse vector field):
https://github.com/deepset-ai/haystack-core-integrations/blob/0cdda5ce6c5e92ef77dcfea046bc7af0bfb47aff/integrations/qdrant/src/haystack_integrations/document_stores/qdrant/migrate_to_sparse.py
